test/mail_test.rb

```diff
@@ -72,6 +72,24 @@ def test_mail_message_alert
     assert(!message.to.empty?)
   end
 
+  def test_mail_message_alert_multiple_emails
+    svc = service(:alert, { :addresses => 'fred@barn.com,test@librato.local' }, alert_payload)
+    message = svc.mail_message
+    assert_equal "[Librato] Alert  has triggered!", message.subject
+    assert_equal ['fred@barn.com', 'test@librato.local'], message.to
+    assert_not_nil message
+    assert(!message.to.empty?)
+  end
+
+  def test_mail_message_alert_multiple_emails_one_invalud
+    svc = service(:alert, { :addresses => 'fred@barn.com,test@@librato.local' }, alert_payload)
+    message = svc.mail_message
+    assert_equal "[Librato] Alert  has triggered!", message.subject
```

Comment:
Yea I believe it is blank if there is no name for the alert (which should never happen). At least that is how it looks in the other tests...